xeon-phi: workaround mlx5_ib issue for card-side ib_get_dma_mr support

Workaround a memory region registration issue due to large (contig) pages.
---
diff -Nrup a/drivers/infiniband/hw/mlx5/mr.c b/drivers/infiniband/hw/mlx5/mr.c
--- a/drivers/infiniband/hw/mlx5/mr.c	2015-11-24 01:14:56.000000000 -0800
+++ b/drivers/infiniband/hw/mlx5/mr.c	2016-02-18 15:41:03.901148829 -0800
@@ -665,7 +665,7 @@ static int get_octo_len(u64 addr, u64 le
 
 static int use_umr(int order)
 {
-	return order <= 17;
+	return order <= 13;
 }
 
 static void prep_umr_reg_wqe(struct ib_pd *pd, struct ib_send_wr *wr,
